cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -436,6 +436,18 @@ def __repr__(self) -> str:
             f'Y:{y_sign}{y.to!s}, Z:{z_sign}{z.to!s})'
         )
 
+    @classmethod
+    def _from_json_dict_(cls, _rotation_map, _inverse_map, **kwargs):
```

Comment:
I wonder if it would be worth having a (parameterized) test in clifford_gate_test.py to run through all the single qubit clifford gates and do the round trip to json and from json and make sure they are equal.  This might be a little bit higher confidence than having the single example in json_test_data, especially since that example has an empty inverse map.

Otherwise seems good to me.

Reply:
Can you please check now?

Reply:
@dstrain115 ping!